docs/contribute/source/os/macos.md

```diff
@@ -32,8 +32,8 @@ WasmEdge will try to use the latest LLVM release to create our nightly build. If
 
 ```bash
 # Tools and libraries
-brew install cmake ninja llvm
-export LLVM_DIR="$(brew --prefix)/opt/llvm/lib/cmake"
+brew install cmake ninja llvm@16
```

Comment:
I think `llvm@19` is fine. The default CI for WasmEdge is using `llvm@19`.

Reply:
It should be `llvm@18`.
https://github.com/WasmEdge/WasmEdge/blob/master/.github/workflows/reusable-build-on-macos.yml#L42